lighty-examples/lighty-community-restconf-netconf-app/src/main/java/io/lighty/examples/controllers/restconfapp/Main.java

```diff
@@ -42,15 +44,26 @@
 public class Main {
 
     private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+    private static final long DEFAULT_TIMEOUT_SECONDS = 30;
 
-    private ShutdownHook shutdownHook;
+    private LightyController lightyController;
+    private SwaggerLighty swagger;
+    private CommunityRestConf restconf;
+    private LightyModule netconfSBPlugin;
+
+    public static void main(String[] args) {
+        Main app = new Main();
+        app.start(args, true);
+    }
 
     public void start() {
         start(new String[] {}, false);
     }
 
+    @SuppressWarnings("IllegalCatch")
     @SuppressFBWarnings("SLF4J_SIGN_ONLY_FORMAT")
     public void start(String[] args, boolean registerShutdownHook) {
+        boolean lightyStartedCorrectly;
```

Comment:
Looks like this is not used anywhere

Reply:
resolved - used when returning true/false from startLighty method